src/main/java/lihua/commands/FindCommand.java

```diff
@@ -0,0 +1,33 @@
+package lihua.commands;
+
+/**
+ * Command class representing a command to find tasks by keyword.
+ */
+public class FindCommand extends Command {
+    /** Command word for find command */
```

Comment:
Do you think it would help if you left spaces between consecutive methods? Have noticed this in a few places and because of the javadoc comments, it may make your code seem a bit "cramped".

Reply:
Sorry correct me if I misunderstood your point🤔. Are you suggesting like I leave spaces of more than one line between consecutive methods to make the code look more 'sparse'?

Reply:
For me, I think the spacing between consecutive methods is good enough. I just think that at the beginning of the class ( between attributes and constructor) you should leave a one line spacing for easier looking